.circleci/config.yml

```diff
@@ -2149,17 +2149,31 @@ workflows:
           docker_image: "308535385114.dkr.ecr.us-east-1.amazonaws.com/caffe2/py2-gcc5-ubuntu16.04:345"
           resource_class: large
       - caffe2_linux_build:
-          name: caffe2_onnx_py3_6_clang7_ubuntu16_04_build
+          name: caffe2_onnx_py3_6_part1_clang7_ubuntu16_04_build
           requires:
             - setup
-          build_environment: "caffe2-onnx-py3.6-clang7-ubuntu16.04-build"
+          build_environment: "caffe2-onnx-py3.6-part1-clang7-ubuntu16.04-build"
           docker_image: "308535385114.dkr.ecr.us-east-1.amazonaws.com/caffe2/py3.6-clang7-ubuntu16.04:345"
       - caffe2_linux_test:
-          name: caffe2_onnx_py3_6_clang7_ubuntu16_04_test
+          name: caffe2_onnx_py3_6_part1_clang7_ubuntu16_04_test
           requires:
             - setup
-            - caffe2_onnx_py3_6_clang7_ubuntu16_04_build
-          build_environment: "caffe2-onnx-py3.6-clang7-ubuntu16.04-test"
+            - caffe2_onnx_py3_6_part1_clang7_ubuntu16_04_build
+          build_environment: "caffe2-onnx-py3.6-part1-clang7-ubuntu16.04-test"
+          docker_image: "308535385114.dkr.ecr.us-east-1.amazonaws.com/caffe2/py3.6-clang7-ubuntu16.04:345"
+          resource_class: large
+      - caffe2_linux_build:
+          name: caffe2_onnx_py3_6_part2_clang7_ubuntu16_04_build
+          requires:
+            - setup
+          build_environment: "caffe2-onnx-py3.6-part2-clang7-ubuntu16.04-build"
+          docker_image: "308535385114.dkr.ecr.us-east-1.amazonaws.com/caffe2/py3.6-clang7-ubuntu16.04:345"
+      - caffe2_linux_test:
+          name: caffe2_onnx_py3_6_part2_clang7_ubuntu16_04_test
+          requires:
+            - setup
+            - caffe2_onnx_py3_6_part2_clang7_ubuntu16_04_build
```

Comment:
caffe2_onnx_py3_6_clang7_ubuntu16_04_build should be enough. :-)

Reply:
Sure. These job are either build + test of build only. And test depends on the build.
But I can change the definition to generate test only jobs.